docs/docs/rest-api/public/api/v2/types/resources.raml

```diff
@@ -0,0 +1,30 @@
+#%RAML 1.0 DataType
+type: object
+description: Resource Allocations
+properties:
+  cpus:
+    displayName: cpu shares
+    description: The number of CPU shares this pod needs per instance. This number does not have to be integer, but can be a fraction.
+    type: number
+    minimum: 0
+    format: float
+    example: 0.1
+  mem:
+    displayName: memory shares
+    type: number
+    minimum: 0
```

Comment:
should probably be 32 since we're only defining resources at the pod level - this is the min memory that mesos needs to launch something